src/Umbraco.Core/Umbraco.Core.csproj

```diff
@@ -43,5 +44,6 @@
         <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
             <_Parameter1>DynamicProxyGenAssembly2</_Parameter1>
         </AssemblyAttribute>
+        <AssemblyAttribute Include="Umbraco.Core.CodeAnnotations.UmbracoSuppressVolatileAttribute"></AssemblyAttribute>
```

Comment:
Was this required, now our own assemblies are marked in the roslyn analyser ?

Reply:
No that's very true, now that they're are marked as allowed, they shouldn't report any errors, I'll remove them again.